app/src/docs/v1.api-spec.yaml

```diff
@@ -0,0 +1,2744 @@
+              $ref: '#/components/schemas/Request-SaveSubmissionDraft'
+      responses:
+        '201':
+          description: Draft submission created successfully
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response-DraftSubmission'
+        "401":
+          $ref: "#/components/responses/Unauthorized"
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        default:
+          $ref: '#/components/responses/Error'
+  /submission/draft/submit:
+    put:
+      summary: Submit a draft submission
+      description: Creates or updates an intake and set status to Submitted
+      operationId: submission-updateDraft
+      tags:
+        - Submission
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Request-SaveSubmissionDraft'
+      responses:
+        '200':
+          description: Draft submission updated successfully
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DB-Submission'
+        "401":
+          $ref: "#/components/responses/Unauthorized"
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        default:
+          $ref: '#/components/responses/Error'
+  /submission/emailConfirmation:
+    put:
+      summary: Send confirmation Email
+      operationId: emailConfirmation
+      tags:
+        - Submission
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/EmailData'
+      responses:
+        '201':
+          description: Email Confirmation was set
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response-Email'
+        "401":
+          $ref: "#/components/responses/Unauthorized"
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        "422":
+          $ref: "#/components/responses/UnprocessableEntity"
+        default:
+          $ref: '#/components/responses/Error'
+  /submission/search:
+    get:
+      summary: Search submissions
+      description: Returns a list of submissions based on the given queries
+      operationId: searchSubmissions
+      tags:
+        - Submission
+      parameters:
+        - $ref: '#/components/parameters/Query-IncludeUser'
+        - in: query
+          name: activityId
+          required: false
+          schema:
+            type: array
+            description: the activity ids to filter on
+            items:
+              $ref: '#/components/parameters/Path-ActivityId'
+        - in: query
+          name: submissionId
+          required: false
+          schema:
+            type: array
+            description: The submissions to filter on
```

Comment:
For consistency, consider either pluralizing activity, or changing submissions to submission IDs.